packages/text-inputs/src/js/react.js

```diff
@@ -58,6 +60,7 @@ AUtextInput.propTypes = {
 	block: PropTypes.bool,
 	status: PropTypes.oneOf([ 'valid', 'invalid' ]),
 	className: PropTypes.string,
+	width: PropTypes.oneOf(["xs", "sm", "md","lg", "xl"])
```

Comment:
```suggestion
	width: PropTypes.oneOf([ 'xs', 'sm', 'md', 'lg', 'xl' ])
```

Try and keep the quote consistent